hammerdb/hammerd_wrapper.py

```diff
@@ -89,12 +89,11 @@ def _summarize_data(data):
               NOPM: {}""".format(entry['nopm']))
         print("+{}+".format("-"*(115)))
 
-def _index_result(server, port, payload):
-    index = "hammerdb-results"
-    es = elasticsearch.Elasticsearch([
-        {'host': server, 'port': port}], send_get_body_as='POST')
+def _index_result(index,server,port,payload):
+    _es_connection_string = str(host) + ':' + str(port)
+    es = elasticsearch.Elasticsearch([_es_connection_string],send_get_body_as='POST')
```

Comment:
If we're going to convert to run_snafu.py then this doesn't matter, but you didn't catch exception here, but you do in run_snafu.

Reply:
This is a temporary fix in place, until we move to run_snafu. hence didn't add the try catch block, also as it happens creating an object will succeed even if given insane credentials so the try catch needs a command like es.info() to check, which again changes depends on the role. given that for our use case, we'll be moving towards run_snafu it didn't make a lot of sense to put all the checks in place here.